src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -7,7 +7,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>$(RoslynPortableTargetFrameworks)</TargetFrameworks>
```

Comment:
@DustinCampbell @tmat Can you check the impact of this?

Reply:
This is just adding another target: `netcoreapp2.0`, so I think that's ok.

Reply:
Yes, that should be fine. I'd be super-concerned if we were adding `netstandard2.0`

Reply:
FWIW, this does represent a *small* concern that it'll signal support for `netcoreapp2.0` while there isn't a good way for customers to use it without MSBuildLocator support. Please, please, please, contribute the code here to the MSBuildLocator. I wouldn't block this PR on that, but we should do the right thing.